File: Python/Math/factorial.py
n = int(input())


def fact(n):
    if n < 0:
        return "Error"
    if n == 0 or n == 1:
        return 1
    factorial = 1
    for i in range(1, n + 1):
        factorial *= i
    return factorial


print(fact(n))

File: JS/1_recursion/factorial.js
function factorial(n) {
    if(n<=1){
        return 1;
    }
    return n * factorial(n-1);
}

console.log("Factorial of 0: "+factorial(0))
console.log("Factorial of 1: "+factorial(1))
console.log("Factorial of 2: "+factorial(2))
console.log("Factorial of 3: "+factorial(3))
console.log("Factorial of 4: "+factorial(4))
console.log("Factorial of 5: "+factorial(5))

